translations/pt-BR/guides/anchor/3-federation-server.md

```diff
@@ -41,8 +41,8 @@ Em seguida, [faça download do servidor federation mais recente](https://github.
 port = 8002
 
 [database]
-type = "mysql" # Ou "postgres" ou "sqlite3"
-dsn = "dbusuario:dbsenha@/internal_accounts"
+type = "postgres" # Ou "sqlite3"
+dsn = "postgres://dbusuario@db____/internal_accounts?sslmode=sslmod"
```

Comment:
I'm not sure what to use for the `dbhost`.

Reply:
@brunomuler or @accordeiro?

Reply:
Also: we should probably make the same change to line 19 as the English version above.

Reply:
> Also: we should probably make the same change to line 19 as the English version above.

@rice2000 2e500c1

Reply:
After speaking with @rice2000 I've dropped the non-English changes in 
1daaa39 because we're not currently using the translated documents, and they will probably need updating in more holistic ways.

Reply:
After speaking with @rice2000 I've dropped the non-English changes in 
1daaa39 because we're not currently using the translated documents, and they will probably need updating in more holistic ways.